websites/G/Goodreads/presence.ts

```diff
@@ -0,0 +1,35 @@
+const presence = new Presence({
+	clientId: "765622778317373440"
+}),
+elapsed = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "logo-1"

```

Comment:
```suggestion
		largeImageKey: 'logo-1'
```

Single quotes to be more consistent.

Reply:
u can use double quotes or even single ones, we are accepting both.